yarn.lock

```diff
@@ -4373,9 +4373,10 @@ extglob@^2.0.4:
     snapdragon "^0.8.1"
     to-regex "^3.0.1"
 
-fake-xml-http-request@^2.0.0:
-  version "2.0.0"
-  resolved "https://registry.yarnpkg.com/fake-xml-http-request/-/fake-xml-http-request-2.0.0.tgz#41a92f0ca539477700cb1dafd2df251d55dac8ff"
+fake-xml-http-request@^2.1.1:
+  version "2.1.1"
+  resolved "https://artifactory.bettermg.com/artifactory/api/npm/npm/fake-xml-http-request/-/fake-xml-http-request-2.1.1.tgz#279fdac235840d7a4dff77d98ec44bce9fc690a6"
```

Comment:
@jcardali it looks like your using an internal registry here which is causing CI to fail

Reply:
@patocallaghan Sorry about that, fixed now